### PR TITLE
[bitnami/external-dns] Update CRDs api version

### LINKED ISSUE #4723: [bitnami/external-dns] Update deprecated "apiextensions.k8s.io/v1beta1" in CRD
**Which chart**:
bitnami/external-dns

**Is your feature request related to a problem? Please describe.**
As described in the Kubernetes [v1.19 release notes](https://v1-19.docs.kubernetes.io/docs/setup/release/notes/#deprecation-4), `apiextensions.k8s.io/v1beta1` is deprecated in favor of `apiextensions.k8s.io/v1` and unavailable from k8s `1.22+`

We use that API version in the [CRD template](https://github.com/bitnami/charts/blob/master/bitnami/external-dns/templates/crd.yaml#L2) in `external-dns`.

**Describe the solution you'd like**
We can update the CRD definition and API version to the new `v1`

**Describe alternatives you've considered**
We can have both CRDs definitions to support both k8s versions `< 1.19` and `1.19+`

**Additional context**
For more information about this deprecation see [kubernetes#82022](https://github.com/kubernetes/kubernetes/issues/82022)

### PR DESCRIPTION
**Pull request by miguelaeh**
**Description of the change**
Update CRD API version

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #4723

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Review by marcosbc — APPROVED** [review]
LGTM

**Review by juan131 — CHANGES_REQUESTED** [review]
Please document the upgrade path in the **README.md**.
I guess that users will have to delete the old CRDs when upgrading from `4.x.x` to `5.x.x`, so we should document that.

File: bitnami/external-dns/README.md [hunk 1]
```diff
@@ -288,6 +288,15 @@ Find more information about how to deal with common errors related to Bitnami’
 
 ## Upgrading
 
+### To 5.0.0
+
+The CRD API version has changed. If you deployed the Helm Chart using `crd.create=true` you need to manually delete the old CRD before upgrading the release.
```
Comment (juan131) [inline bitnami/external-dns/README.md:293]:
```suggestion
The CRD were updated according to the latest changes in the upstream project. As a consequence, the CRD API version was moved from `apiextensions.k8s.io/v1beta1` to `apiextensions.k8s.io/v1`.

If you deployed the Helm Chart using `crd.create=true` you need to manually delete the old CRD before upgrading the release as shown below:
```

**Review by juan131 — APPROVED** [review]
LGTM with suggestion to improve the README.md

**Pull request merged**